src/components/Deporte.js

```diff
@@ -1,7 +1,10 @@
 import React from "react";
```

Comment:
Creo que tes os 2 PR mixturados 😱😉.

Temos que pasar á versión antigua de `Deporte.js`. O mais sinxelo é recuperar a liña borrada `<h2>{this.props.sportEvents.title}</h2>`.